deploy/cdk/lib/elasticsearch/elastic-stack.ts

```diff
@@ -10,13 +10,14 @@ export interface ElasticStackProps extends cdk.StackProps {
 
 export class ElasticStack extends cdk.Stack {
   readonly domainName: string
+  readonly domain: CfnDomain
 
   constructor(scope: cdk.Construct, id: string, props: ElasticStackProps) {
     super(scope, id, props)
     this.domainName = `${props.namespace}-sites`
```

Comment:
FYI, this will also need a TODO to not specify the domainName and let cloudformation create the name - like the manifest lambda stack.